plugins/woocommerce/client/blocks/packages/components/checkbox-control/index.tsx

```diff
@@ -85,7 +85,6 @@ export const CheckboxControl = forwardRef<
 					<svg
 						className="wc-block-components-checkbox__mark"
 						aria-hidden="true"
-						xmlns="http://www.w3.org/2000/svg"
```

Comment:
Noticed it while doing style changes to the component. For inline SVGs in React this attribute is not needed.